include/seqan3/range/views/pairwise_combine.hpp

```diff
@@ -543,7 +543,7 @@ class pairwise_combine_view : public std::ranges::view_interface<pairwise_combin
      */
     template <typename other_range_t>
     //!\cond
-        requires !std::same_as<remove_cvref_t<other_range_t>, pairwise_combine_view> &&
+        requires (!std::same_as<remove_cvref_t<other_range_t>, pairwise_combine_view>) &&
                  std::ranges::viewable_range<other_range_t> &&  // Must come after self type check to avoid conflicts with the move constructor.
```

Comment:
146 character in one line: Clearly to much, but is a preexisting issue anyway.
```suggestion
                  std::ranges::viewable_range<other_range_t> &&  // Must come after self type check to avoid conflicts with the move constructor.
```